lib/hammer_cli_csv/roles.rb

```diff
@@ -84,9 +84,7 @@ def create_roles_from_csv(line)
             @existing_roles[name] = role['id']
           else
             print "Updating role '#{name}'..." if option_verbose?
-            @api.resource(:roles).call(:update, {
-                'id' => @existing_roles[name]
-            })
+            # Nothing to update
```

Comment:
i'm referring to the line above this. `print "Updating role ..."` Why are you printing that if you aren't updating the role?